shared/teams/main/no-teams-placeholder.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+import * as Container from '../../util/container'
+import * as Kb from '../../common-adapters'
+import {teamsLoadedWaitingKey} from '../../constants/teams'
+import * as RouteTreeGen from '../../actions/route-tree-gen'
+import {globalMargins, desktopStyles, platformStyles, styleSheetCreate} from '../../styles'
```

Comment:
nit: `* as Styles`

Reply:
that was a holdover line from the previous `beta-note.tsx` I must have missed 😄